express-api/src/services/notifications/notificationServices.ts

```diff
@@ -442,6 +445,12 @@ const convertChesStatusToNotificationStatus = (chesStatus: string): Notification
       return NotificationStatus.Failed;
     case 'completed':
       return NotificationStatus.Completed;
+    case '404':
+      return NotificationStatus.NotFound;
+    case '422':
+      return NotificationStatus.Invalid;
+    case '401':
+      return NotificationStatus.Unauthorized;
```

Comment:
Do we need statuses for these? It seems like they describe the request, not the state of the message.